packages/nodes-base/credentials/Aws.credentials.ts

```diff
@@ -285,7 +285,7 @@ export class Aws implements ICredentialType {
 		const method = requestOptions.method;
 		let body = requestOptions.body;
 		let region = credentials.region;
-		const query = requestOptions.qs?.query as IDataObject;
+		let query = requestOptions.qs?.query as IDataObject;
```

Comment:
should we update this code to either use `requestOptions.qs?.query` if the JSON object is set, or to create the query object based on rest of the values in `requestWithUri.qs`, since the UI only allows only one of the options.